cmake/compilers/Clang.cmake

```diff
@@ -13,12 +13,16 @@
 # limitations under the License.
 
 if (EMSCRIPTEN)
-  set(TBB_EMSCRIPTEN 1)
-  set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -fexceptions)
-  set(TBB_TEST_LINK_FLAGS  ${TBB_COMMON_LINK_FLAGS} -fexceptions -sINITIAL_MEMORY=65536000 -sALLOW_MEMORY_GROWTH=1 -sEXIT_RUNTIME=1)
-  if (NOT EMSCRIPTEN_WITHOUT_PTHREAD)
-      set_property(TARGET Threads::Threads PROPERTY INTERFACE_LINK_LIBRARIES "-pthread")
-  endif()
+    set(TBB_EMSCRIPTEN 1)
+    set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -fexceptions)
+    set(TBB_TEST_LINK_FLAGS  ${TBB_COMMON_LINK_FLAGS} -fexceptions -sINITIAL_MEMORY=65536000 -sALLOW_MEMORY_GROWTH=1 -sMALLOC=mimalloc -sEXIT_RUNTIME=1)
+    if (NOT EMSCRIPTEN_WITHOUT_PTHREAD)
+        set_property(TARGET Threads::Threads PROPERTY INTERFACE_LINK_LIBRARIES "-pthread")
+    endif()
+    set(TBB_EMSCRIPTEN_STACK_SIZE 5242880)
```

Comment:
Honestly, I'm not sure if it is an optimal stack size.

Reply:
Is it 64KB? (https://github.com/emscripten-core/emscripten/pull/18191/files)

Reply:
Hmm, that an interesting point. Ideally TBB should propagate `TOTAL_STACK` if it was set by user application.
But you are right the default should be changed to 64kb.